packages/astro/tsconfigs/base.json

```diff
@@ -17,6 +17,11 @@
     // Properly support importing CJS modules in ESM
     "esModuleInterop": true,
     // Skip typechecking libraries and .d.ts files
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    // Add alias for assets folder for easy reference to assets
+    "baseUrl": ".",
+    "paths": {
+      "~/assets/*": ["src/assets/*"]
```

Comment:
Has there been much discussion about this? How does this merge with user-defined aliases? I notice we don’t show use of this at all in the new Assets docs.

Reply:
I haven't received a single piece of feedback on this, I'll admit 🤷‍♀️ I don't necessarily like aliases myself, but I get the usage in Markdown (where it can be a bit cumbersome to know where `src/assets` is)

Reply:
I definitely understand the convenience — we have `src/` aliased in docs to allow easy referencing of components etc. from MDX — but just wasn’t sure about setting a default if we don’t for anything else. People have opinions on how these should look, and if we haven’t documented so far, maybe this is not required as a core feature. That said, if it went through the RFC process and people think it’s important, that’s cool!

Reply:
Definitely something we can look at post release, we have until 3.0 to figure out the alias 😄